ConsentViewController/Classes/SPConsentManager.swift

```diff
@@ -417,6 +434,14 @@ import UIKit
             }
         }
     }
+
+    public func loadCCPAPrivacyManagerChildPM(withFallbackId id: String, tab: SPPrivacyManagerTab = .Default) {
```

Comment:
@Nevazhnovu should we rename this method to `loadCCPAGroupPrivacyManager(...)` ?
The developer doesn't (and shouldn't) know about the existence of a "child pm id". For our user, they are loading a group pm and not a child pm. 

Same for GDPR below.